### ISSUE TITLE
Can't compile hello world with colecodev

### ISSUE THREAD

**Issue by bkumanchik** [alekmaul/pvcollib#6]
Can't compile hello world with colecodev, I'm getting:

> "make" 
Makefile:39: *** empty variable name.  Stop.

> Process Exit Code: 2
> Time Taken: 00:00

Thanks for any help

**Comment by alekmaul** [comment]
Sam reply as your post for pvsneslib,
-are you sure you read https://github.com/alekmaul/pvcollib/wiki/Installation-and-helloworld-example
- how do you tried to compile it, with an IDE, in a CLI?

**Comment by bkumanchik** [comment]
Yes, (sorry, it's me again) when I first try to clean or make I get:

**Makefile:39: *** empty variable name.  Stop.**

Then I checked line 39 of **Makefile** and it's:

**LIBOBJS +:=**	

So I removed the "+" and now clean works and make almost works except for:

**make: *** [/c/colecodev/devkitcol/col_rules:47: /c/colecodev/coleco-examples/helloworld/helloworld.rom] Error 1**

**Comment by bkumanchik** [comment]
Am I missing a system variable that would find **col_rules** ?

**Comment by bkumanchik** [comment]
the full output of make:
<img width="659" alt="Screenshot 2024-02-16 121759" src="https://github.com/alekmaul/pvcollib/assets/25464489/09d490d2-57f1-479c-beef-450ab36ed080">

**Comment by bkumanchik** [comment]
and the helloworld folder after clean then make:
<img width="814" alt="Screenshot 2024-02-16 121906" src="https://github.com/alekmaul/pvcollib/assets/25464489/c3b71c8c-e034-4f72-bd14-50c2092f18b5">

**Comment by bkumanchik** [comment]
one last thing, dir matches **Makefile**:
<img width="671" alt="Screenshot 2024-02-16 122050" src="https://github.com/alekmaul/pvcollib/assets/25464489/ed32df56-b47e-4234-b944-cb3539c650c7">

**Comment by bkumanchik** [comment]
I get same error with IDE and CLI

**Comment by alekmaul** [comment]
OK, I will check the installation. It is a long time that I did not work on colecodev. Perhaps it is now not OK with Windows 11.

**Comment by bkumanchik** [comment]
Thanks Alek! :)

**Comment by bkumanchik** [comment]
Hey Alek, let me know if there's something else I can do to test this.

**Comment by alekmaul** [comment]
I'm sorry but i'm currently on SNES to prepare a french convention next week. I have no time to check my coleco devkit.
I will check it mi march if everything goes well.

**Comment by bkumanchik** [comment]
No problem, just checking, good luck with the convention, you do awesome work! 
    On Sunday, February 25, 2024 at 02:26:39 AM EST, alekmaul ***@***.***> wrote:  
 
 


I'm sorry but i'm currently on SNES to prepare a french convention next week. I have no time to check my coleco devkit.
I will check it mi march if everything goes well.

**Comment by bkumanchik** [comment]
Hope your convention went well :)

**Comment by alekmaul** [comment]
yes it was.
You must know that I have no time to work on the CV devkit currently, i'm alone on this project and it is not easy to have time for both SNES and CV.
Sorry

**Comment by bkumanchik** [comment]
I do, sorry, I will never bother you again.
    On Wednesday, April 3, 2024 at 11:23:40 PM EDT, alekmaul ***@***.***> wrote:  
 
 


yes it was.
You must know that I have no time to work on the CV devkit currently, i'm alone on this project and it is not easy to have time for both SNES and CV.
Sorry
